test_runtime.sh

```diff
@@ -78,7 +78,7 @@ tar -xf  rootfs.tar.gz -C ${TESTDIR}
 cp runtimetest ${TESTDIR}
 
 pushd $TESTDIR > /dev/null
-ocitools generate --output config.json "${TEST_ARGS[@]}" --rootfs '.'
+ocitools generate --tty --output config.json "${TEST_ARGS[@]}" --rootfs '.'
```

Comment:
On Mon, Jul 18, 2016 at 03:54:39PM -0700, Mrunal Patel wrote:
> > @@ -78,7 +78,7 @@ tar -xf  rootfs.tar.gz -C ${TESTDIR}
> >  cp runtimetest ${TESTDIR}
> >  
> >  pushd $TESTDIR > /dev/null
> > -ocitools generate --output config.json "${TEST_ARGS[@]}" --rootfs '.'
> > +ocitools generate --tty --output config.json "${TEST_ARGS[@]}" --rootfs '.'
> 
> @wking I have created a runtime-spec PR https://github.com/opencontainers/runtime-spec/pull/518

+1.  In the meantime, can we land this PR without this change?